setup.py

```diff
@@ -52,6 +52,7 @@
         "google-cloud-documentai >= 1.2.1, < 3.0.0dev",
         "google-cloud-storage >= 1.31.0, < 3.0.0dev",
         "numpy >= 1.18.1",
+        "pikepdf >= 6.2.9, < 8.0.0",
```

Comment:
Why did you use pikepdf over PyPDF2 ?

Reply:
This is the library that Matthew Carroll used for his original sample.
From my understanding, he went through a vetting process to decide what to use. https://github.com/GoogleCloudPlatform/document-ai-samples/tree/main/pdf-splitter-python